### DISCUSSION TITLE
Downgrading to a specific Mingw version

### DISCUSSION THREAD

**Discussion by anoopkumares** [msys2/MINGW-packages#8306] [Q&A]
I have MSYS2 64 bit installed in my system.
When I try to check the gcc version, I am getting the following output.
```
**$ gcc --version
gcc.exe (Rev9, Built by MSYS2 project) 10.2.0
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.**
```
But I want to use a specific mingw version for my project which uses qt library.
When I check my build log and installation folder of my qt project I found this.

**|Project name: DataManager
|Project version: undefined
|C++ compiler for the host machine: ccache c++ (gcc 4.9.2 "c++ (i686-posix-dwarf-rev1, Built by MinGW-W64 project) 4.9.2")
|C++ linker for the host machine: c++ ld.bfd 2.24
|Program python3 found: YES
|Found qmake: C:\Qt\5.6.3\mingw49_32\bin\qmake.EXE (5.6.3)**

I am assuming that it uses mingw49_32 version and gcc version 4.9.2.
I would like to use the same gcc version (4.9.2) in MSYS2.

How can I achieve it ?

**Comment by Biswa96** [comment]
You can downgrade a specific package by downloading the .tar.zst file from a Pacman repo like this http://repo.msys2.org/mingw/ and install it with `pacman -U` command. More details here https://wiki.archlinux.org/index.php/downgrading_packages.

But I didn't get the actual scenario for doing that. That qmake path doesn't belong to msys2 installation and we've already a qt5 and qt6 packages which can be used/linked with a project.
